### DISCUSSION TITLE
bhm-zone-display

### DISCUSSION THREAD

**Discussion by agaertner** [agaertner/bhm-zone-display#38] [bhm-zone-display]
### Regions of Tyria

A beautiful area notification as you walk in and out of different zones!

[!(Preview)](https://github.com/agaertner/bhm-zone-display/assets/13819164/882f5b9d-d6f1-4159-8de3-2e0d0e1575a9.mp4)

**Comment by ragmats** [comment]
This is a great addon that adds a beautiful visual flare to the game, but because of the small size of the zones, I feel it happens way too frequently and I have sadly turned it off even though I like it a lot. I wish it had the option to limit showing the text only the first occasion of any major zone (i.e. Lion's Arch, Bjora Marches, etc.), and then ignore the rest of the smaller zones (unless desired). And/or perhaps the ability to set a frequency limiter, i.e. don't show more than x times per minute, etc., similar to how it can be disabled during combat, but with more flexible customization.
